observation_portal/requestgroups/models.py

```diff
@@ -330,6 +331,19 @@ def get_remaining_duration(self, configurations_after_priority):
             start_time,
             configurations_after_priority
         )
+        if include_current:
```

Comment:
In this case where we want to calculate end time based on a configuration statuses exposure start time, we want to add the remaining configuration status duration to the current configurations duration of exposures (duration minus front_padding).